__tests__/hooks/useEtherspotHistory.test.js

```diff
@@ -0,0 +1,63 @@
+import { renderHook, waitFor } from '@testing-library/react';
+
+// hooks
+import { useEtherspotHistory, EtherspotUi } from '../../src';
+
+describe('useEtherspotHistory()', () => {
+  it('getAccountTransactions() returns account history', async () => {
+    const wrapper = ({ children }) => (
+      <EtherspotUi provider={null}>
+        {children}
+      </EtherspotUi>
```

Comment:
Should this be `EtherspotTransactionKit`?

Reply:
Oops sorry, was not updated to master, fixed!